sympy/testing/pytest.py

```diff
@@ -214,6 +214,19 @@ def warns(warningcls, *, match=''):
             raise Failed(msg)
 
 
+def _both_exp_pow(func):
```

Comment:
Can you add a comment here explaining why this is here and that it is only temporary while this fix is being done.